lib/pure/net.nim

```diff
@@ -241,7 +265,72 @@ when defined(ssl):
 
     discard newCTX.SSLCTXSetMode(SSL_MODE_AUTO_RETRY)
     newCTX.loadCertificates(certFile, keyFile)
-    return SSLContext(newCTX)
+
+    result = SSLContext(newCTX)
+    # this is never freed, but SSLContext can't be freed anyway yet
+    let extraInternal = new(SslContextExtraInternal)
+    GC_ref(extraInternal)
```

Comment:
Ok, I will add destructor, this comment only means that there already is a memory leak involving SSLContexts.